hugolib/multilingual.go

```diff
@@ -0,0 +1,48 @@
+package hugolib
+
+import (
+	"github.com/spf13/cast"
+	"github.com/spf13/viper"
+)
+
+type Multilingual struct {
+	enabled bool
+	config  *viper.Viper
+
+	Languages []string
+}
+
+func (ml *Multilingual) GetString(key string) string { return cast.ToString(ml.Get(key)) }
+func (ml *Multilingual) GetStringMap(key string) map[string]interface{} {
+	return cast.ToStringMap(ml.Get(key))
+}
+
+func (ml *Multilingual) GetStringMapString(key string) map[string]string {
+	return cast.ToStringMapString(ml.Get(key))
+}
+
+func (ml *Multilingual) Get(key string) interface{} {
```

Comment:
It isn't clear to me what is part of the "multilingual config" and not. I tried to port my own blog to be multilingual, and intuitively put paginatePath into the multilingual section expecting that to just work (I notice the viper.Get("...") so I understand why it doesn't). I think all config that is used during build of a site should be per language.